### DISCUSSION TITLE
bidir option does not work when using UDP mode -- iperf 3.7

### DISCUSSION THREAD

**Discussion by worktest8888** [esnet/iperf#1382] [General]
Hi 
I use iperf 3.7 with --bidir flag to do some simulate testing, i find in UDP mode  client and server will be freeze, but in TCP mode, it works fine
Does it means iperf3.7 not support --bidir in UDP mode

**Comment by davidBar-On** [comment]
Hi, this seem to be a similar issue to  #991. According to [my comment](https://github.com/esnet/iperf/issues/991#issuecomment-629617140) to that issue, I had the same problem 2 years ago.  However, when I run the same 3.7 version in the same environment, UDP with `--bidir` is successful.  I am not sure what was changed.

Can you add more details about the exact command lines you are using for the server and the client (IP addresses may be removed), and the environment you are using for both?  Also, can you run both client and server with the `-d -V` options (in addition to the `-u --bidir`) and send their outputs?
